.github/workflows/mac_build_test.yml

```diff
@@ -47,7 +47,7 @@ jobs:
       - name: Initial setup
         shell: bash -l {0}
         run: |
-          brew install eigen gcc@6 hdf5
+          brew install eigen hdf5
```

Comment:
I didn't see `gcc@6` removed in the original PR. Can I ask why it was removed here?